docusaurus.config.js

```diff
@@ -82,6 +82,8 @@ module.exports = {
                 },
             ],
         },
+        // Image used for og:image meta tag
```

Comment:
I'd never seen this before, and it took some digging to figure out what it was . Could we add a link to this site to the comment  for clarity? https://ogp.me/
```suggestion
        // Image used for og:image meta tag - see https://ogp.me/
```